backend/hitas/templates/unconfirmed_maximum_price.jinja

```diff
@@ -0,0 +1,113 @@
+{% extends "components/base.jinja" %}
```

Comment:
I would love to see some kind of tests to these. While testing PDFs is far from optional, these templates contain a bit of logic and tests would help not to break these. These are currently untested (in sense of the content that they have).

There's many ways to test these but one naive and fast wound be something like:
```
test__render_unconfirmed_max_price__surface_are_price_ceiling():
   context = {...}
   html = render_to_pdf("unconfirmed_maximum_price.jinja", context)
   assert "rajahinnan perusteella" in html
  render
test_            {% if prices.construction_price_index.value > prices.market_price_index.value %}
                <td>Hitas-huoneiston velaton enimmäishinta rakennuskustannusindeksillä laskettuna, euroa</td>
```